src/components/Header/Header.jsx

```diff
@@ -2,36 +2,68 @@ import React, { useState } from "react";
 import { Link } from 'react-router-dom';
 import classNames from "classnames";
 import icon_close from "../../assets/icon-close.svg";
+import icon_hamburger from "../../assets/icon-hamburger.svg";
 import logo from "../../assets/logo.svg";
 import styles from "./Header.module.css";
 
 const Header = () => {
-  const [mobileMenu, setMobileMenu] = useState(false);
+  const [mobileMenuOpen, setMobileMenuOpen] = useState(false);
   const toggleMenu = () => {
-    setMobileMenu((toggleMenu) => !toggleMenu);
+    setMobileMenuOpen(!mobileMenuOpen);
```

Comment:
I ended up changing the name of the state and its setter, but the important thing I changed here is that **it's just providing the new value** (the opposite boolean value), not running a function (and especially not running a function that returns `void`)